package.json

```diff
@@ -93,7 +94,6 @@
   },
   "dependencies": {
     "@types/js-yaml": "^4.0.5",
```

Comment:
you can move @types/js-yaml also to be a dev dependency as it is not exposed at all.

Reply:
![image](https://github.com/probot/octokit-plugin-config/assets/5059100/e197604a-20c8-49f9-8fc5-a4ef733dfe34)

Reply:
Thanks, fixed!